containers/sarek/environment.yml

```diff
@@ -0,0 +1,24 @@
+# You can use this file to create a conda environment for this pipeline:
+# conda env create -f environment.yml
+name: sarek-core
```

Comment:
We should specify a version here
so I would go for `sarek-core-dev` or `sarek-core-2.1`

Reply:
can we just leave as `sarek` ?